File: group-projects/2021-03_media_bazaar/MediaBazaar Website/classes/Employee.php
<?php
class Employee extends Database {
    private $id;
    private $first_name;
    private $last_name;
    private $birth_date;
    private $email;
    private $password;
    private $job_position;
    private $phone_number;
    private $address;
    private $salary;
    private $gender;
    private $education;
    private $contract;
    private $days_off;
    private $contract_hours;
    private $notes;

    public function __construct($id, $first_name, $last_name, $birth_date, $email, $password, $job_position, $phone_number, $address, $salary, $gender, $education, $contract, $days_off, $contract_hours, $notes) {
        $this->id = $id;
        $this->first_name = $first_name;
        $this->last_name = $last_name;
        $this->birth_date = $birth_date;
        $this->email = $email;
        $this->password = $password;
        $this->job_position = $job_position;
        $this->phone_number = $phone_number;
        $this->address = $address;
        $this->salary = $salary;
        $this->gender = $gender;
        $this->education = $education;
        $this->contract = $contract;
        $this->days_off = $days_off;
        $this->contract_hours = $contract_hours;
        $this->notes = $notes;
    }

    public function getId() {
        return $this->id;
    }

    public function getFirstName() {
        return $this->first_name;
    }

    public function getLastName() {
        return $this->last_name;
    }

    public function getGender() {
        return $this->gender;
    }

    public function getBirthDate() {
        return $this->birth_date;
    }

    public function getEmail() {
        return $this->email;
    }

    public function getPassword() {
        return $this->password;
    }

    public function getJobPosition() {
        return $this->job_position;
    }

    public function setFirstName($firstName) {
        $this->first_name = $firstName;
    }

    public function setLastName($lastName) {
        $this->last_name = $lastName;
    }

    public function setPassword($password) {
        $this->password = $password;
    }
}

File: group-projects/2021-03_media_bazaar/MediaBazaar Windows Application/PRJMediaBazaar/Logic/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PRJMediaBazaar.Logic
{
    public class Employee
    {
        
        public Employee(int id, string firstName, string lastName, DateTime birthDate,
           string gender, double salary, string email, string password,
          string jobPosition, int phoneNumber, string address, string education, string contract,
            int daysOff, int contractHours, int daysOffLeft)
        {
            Id = id;
            FirstName = firstName;
            LastName = lastName;
            BirthDate = birthDate;
            Gender = gender;
            Salary = salary;
            Email = email;
            Password = password;
            JobPosition = jobPosition;
            PhoneNumber = phoneNumber;
            Address = address;
            Education = education;
            Contract = contract;
            DaysOff = daysOff;
            ContractHours = contractHours;
            DaysOffLeft = daysOffLeft;
        }

        public Employee(int id, string firstName, string lastName, string jobPosition)
        {
            Id = id;
            FirstName = firstName;
            LastName = lastName;
            JobPosition = jobPosition;
        }

        public int Id { get; private set; }
        public string FirstName { get; private set; }
        public string LastName { get; private set; }
        public DateTime BirthDate { get; private set; }
        public string Gender { get; private set; }
        public double Salary { get; private set; }
        public string Email { get; private set; }
        public string Password { get; private set; }
        public string JobPosition { get; private set; }
        public int PhoneNumber { get; private set; }
        public string Address { get; private set; }
        public string Education { get; private set; }
        public string FullName { get { return FirstName + " " + LastName; } }
        public String Contract { get; set; }
        public int DaysOff { get; set; }
        public int ContractHours { get; set; }
        public int DaysOffLeft { get; set; }
        public String Note { get; set; }       
    }
}
